src/app/app.cpp

```diff
@@ -44,6 +44,8 @@
 using namespace mu::app;
 using namespace mu::appshell;
 
+int g_jackTransportDelay;
```

Comment:
This seems unused at the moment

Reply:
g_ means global, its use is tucked away in jackaudiodriver.cpp

Reply:
Ah sorry, apparently that file was collapsed so Ctrl+F didn't find it :)
However, global variables that are referenced across multiple files (even multiple modules) don't seem great. It'd probably be better to put this in something like `AudioConfiguration`.